src/prog/main.F90

```diff
@@ -1385,7 +1385,18 @@ subroutine parseArguments(env, args, inputFile, paramFile, accuracy, lgrad, &
          else
             call env%error("No color scheme provided for --color option", source)
          end if
-      
+
+
+      case('--spinpol')
+         if (get_xtb_feature('tblite')) then
+            call env%warning("Employing tblite library for spin-polarization", source)
+            call set_exttyp('tblite')
```

Comment:
We should only raise a warning if ``--tblite`` is not manually specified. Maybe check after evaluating all arguments if ``--spinpol`` has been set without using ``tblite``. Then you can also write a more specified warning (i.e. suggesting the user to explicitly order ``tblite`` to be used)

Reply:
Sorry, I had to transfer your check from the `newCalculator` subroutine to the `xtbMain`. Apparently, this subroutine is used for the **iff** and **docking** unit tests, and could not be modified in such way.

Reply:
My bad, alternatively one could have checked if the optional tblite_input is present.